src/main/java/gov/cabinetofice/gapuserservice/model/BlacklistedToken.java

```diff
@@ -0,0 +1,28 @@
+package gov.cabinetofice.gapuserservice.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@Table(name = "token_blacklist", indexes = @Index(columnList = "token"))
+public class BlacklistedToken {
+
+    @Id
+    @GeneratedValue
+    private int id;
+
+    @Column(length = 4000)
```

Comment:
feel like this is sufficient but input appreciated.

Reply:
From some googling sounds like 8kb is the max JWT size. Though technically we could just store the hash (signature) here which would save on some space... Would also mean we aren't storing any of the JWTs payload which might be a security plus